src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
@mugling Imagine you have more than one container for the objects (which is quite likely). In this case you have to refactor these static member functions out of the class. This signals that they shouldn't belong there. In other words, the class makes assumptions that it shouldn't make.